x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -604,13 +606,23 @@ export class Plugin implements ISecuritySolutionPlugin {
       this.telemetryReceiver
     );
 
-    plugins.fleet?.fleetSetupCompleted().then(() => {
-      if (plugins.taskManager) {
-        this.checkMetadataTransformsTask?.start({
-          taskManager: plugins.taskManager,
-        });
+    const endpointPkgInstallationPromise = this.endpointContext.service
+      .getInternalFleetServices()
+      .packages.getInstallation(FLEET_ENDPOINT_PACKAGE);
+    Promise.all([endpointPkgInstallationPromise, plugins.fleet?.fleetSetupCompleted()]).then(
+      ([endpointPkgInstallation]) => {
+        if (plugins.taskManager) {
+          if (
+            endpointPkgInstallation?.version &&
+            isEndpointPackageV2(endpointPkgInstallation.version)
```

Comment:
a future todo, maybe we can get to it this release cycle.. is probably renaming this function.

We're not actually checking if the package _manifest_ is set to v2+, IIRC, it's just checking the package is 8.12. 
And for the unattended flag here, it's not actually related to being at manifest v2, we just happened to add that setting once we were at manifest v2 again. 

might be better as `isEndpointPackageVersionAtLeast(endpointPkgInstallation.version, '8.12')`.